ui/app/AppLayouts/Onboarding2/OnboardingFlow.qml

```diff
@@ -232,6 +238,7 @@ OnboardingStackView {
 
         HelpUsImproveStatusPage {
             onPrivacyPolicyRequested: d.openPrivacyPolicyPopup()
+            onShareUsageDataRequested: d.seenUsageDataPrompt = true
```

Comment:
Does this mean that only when the user accepts sharing metrics we set `seenUsageDataPrompt`? Is that's so, I think it's not according to the issue.
> So if they select Create profile then opt in or out of analytics and progress to the create profile menu, when they click back, they should be taken back to the initial onboarding screen and not back to the analytics screen.

Reply:
No, in any case; the signal is emitted with a bool param that's not handled here at all